doc/source/user/c-info.ufunc-tutorial.rst

```diff
@@ -1048,133 +1048,6 @@ The C file is given below.
         #endif
         }
 
-
```

Comment:
Duplicate?

Reply:
I was curious about the deletion that starts here.

Reply:
Yes, duplicate